use/2015-12-15-microsatellite-differentiation.Rmd

```diff
@@ -0,0 +1,284 @@
+---
+title: "Calculating genetic differentiation from SSR data"
+---
+
+# Introduction
+
+This workflow will show you how to assess population differentiation with
+microsatellite/SSR data. We will analyse F-statistics, AMOVA, and Discriminant
+Analysis of Principal Components (DAPC). We will analyze a data set of bovine 
+samples from France and West Africa. We want to know if differentiation exists 
+between Breeds or if it's based on geographic separation.
+
+# Data
+
+We are going to use the `microbov` data set from the *adegenet* package. This 
+data set consists of 704 bovine samples over 30 microsatellite loci (Laloe et 
+al., 2007). This data set also contains 
```

Comment:
Convert to : "We will also need information about the Species, Breed, and Country/region of origin for each sample. That data is stored in the file called [`bovine_strata.csv` (click
here to download)](https://github.com/NESCent/popgenInfo/tree/master/data/bovine_strata.csv)."